Machine-Learning/README.md

```diff
@@ -0,0 +1,106 @@
+- [Deep Learning basics with Python, TensorFlow and Keras -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDfhTox0AjmQ6tvTgMBZBEXN)
+- [Deep Learning Crash Course -FreeCodeCamp](https://youtu.be/VyWAvY2CF9c)
+- [Keras with TensorFlow Course - Python Deep Learning and Neural Networks for Beginners Tutorial -FreeCodeCamp](https://youtu.be/qFJeN9V1ZsI)
+- [Neural Networks from Scratch in Python -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDcjD5BAw2DxE6OF2tius3V3)
+- [Practicle Deep Learning by Fast.ai -FreeCodeCamp](https://youtu.be/0oyCUWLL_fU)
+- [Neural Network Full Course](https://youtu.be/ob1yS9g-Zcs)
+- [ Convolutional Neural Networks for Visual Recognition -Standford(Youtube)](https://youtube.com/playlist?list=PL3FW7Lu3i5JvHM8ljYj-zLfQRF3EO8sYv)
+- [Deep Learning -by Standford online](https://youtube.com/playlist?list=PLoROMvodv4rOABXSygHTsbvUz4G_YQhOb)
+- [Deep Learning with PyTorch (9-Day Mini-Course)](https://machinelearningmastery.com/deep-learning-with-pytorch-9-day-mini-course/)
+
+## Spacelization in Machine Learning
+
+### Natural Language Processing
+- [Natural Language Processing with TensorFlow2 -by Freecodecamp](https://youtu.be/B2q5cRJvqI8)
+- [Natural Language Processing with Spacy and Python -By FreeCodeCamp](https://youtu.be/dIUTsFT2MeQ)
+- [Natural Language Processing with Python -Sentdex](https://youtube.com/playlist?list=PLQVvvaa0QuDf2JswnfiGkliBInZnIC4HL)
+- [Distributed Representations of Words and Phrases and their Compositionality" by Tomas Mikolov- Research Paper](https://proceedings.neurips.cc/paper/2013/file/9aa42b31882ec039965f3c4923ce901b-Paper.pdf)
+- [Natural Language Processing with Python by Steven Bird, Ewan Klein, and Edward Loper -Book pdf](https://tjzhifei.github.io/resources/NLTK.pdf)
+
+### Computer Vision and Image Processing
+-[Open CV course with Python -by FreeCodeCamp](https://youtu.be/oXlwWbU8l2o)
+-[Image processing tutorials with Python -by Edureka](https://youtu.be/sfheWK72L74)
+-[Computer vision algorithms and applications by richard szeliski -Book pdf](https://www.cs.ccu.edu.tw/~damon/tmp/SzeliskiBook_20100903_draft.pdf)
+-[ImageNet Classification with Deep Convolutional Neural Networks by Alex Krizhevsk -Research Paper](https://papers.nips.cc/paper_files/paper/2012/file/c399862d3b9d6b76c8436e924a68c45b-Paper.pdf)
+
+### Reinforcement Learning
+- [Reinforcement Learning in 3 hours -by Nicholas Renotte](https://youtu.be/Mut_u40Sqz4)
+- [Reinforcement Learning -by Standford](https://youtube.com/playlist?list=PLoROMvodv4rOSOPzutgyCTapiGlY2Nd8u)
+- [Reinforcement Learning full course -by FreeCodeCamp](https://youtu.be/ELE2_Mftqoc)
+- [Reinforcement Learning: An Introduction  by Richard S. Sutton and Andrew G. Barto -book pdf](https://web.stanford.edu/class/psych209/Readings/SuttonBartoIPRLBook2ndEd.pdf)
+- [Playing Atari with Deep Reinforcement Learning by Volodymyr Mnih et al -Research Paper](https://www.cs.toronto.edu/~vmnih/docs/dqn.pdf)
+
+## Practice Project
```

Comment:
Other projects that are more applicable in real life:

- https://pyimagesearch.com/2016/08/10/imagenet-classification-with-python-and-keras/
- https://pyimagesearch.com/2016/10/03/bubble-sheet-multiple-choice-scanner-and-test-grader-using-omr-python-and-opencv/
- https://www.youtube.com/playlist?list=PLzMcBGfZo4-lwGZWXz5Qgta_YNX3_vLS2
- https://pyimagesearch.com/2019/02/18/breast-cancer-classification-with-keras-and-deep-learning/